CUDA.gitignore

```diff
@@ -4,3 +4,4 @@
 *.ptx
 *.cubin
 *.fatbin
+*.*
```

Comment:
@fgragpp this rule seems excessive, and without more context I plan to close this out -  are you able to elaborate?